Dockerfile.alpine

```diff
@@ -147,6 +147,12 @@ RUN cd postgresql-16.* && \
 # Use the PostgreSQL Alpine image as our output image base
 FROM postgres:${PGTARGET}-alpine3.20
 
+# Copy default configuration in case the original container does not provide it (Bitnami ...)
+RUN mkdir -p /opt/pgautoupgrade && \
+  chmod 660 /opt/pgautoupgrade && \
+  chown 999:999 /opt/pgautoupgrade
+COPY --chown=999 postgresql.conf pg_hba.conf /opt/pgautoupgrade/
```

Comment:
as an alternative to the copy process, we could also invoke `pg_setup_hba_conf()` in the `entrypoint.sh` to generate a valid `pg_hba.conf`. `postgresql.conf` I am not sure ...